src/main/scala/rps/repository/GameRepository.scala

```diff
@@ -0,0 +1,17 @@
+package rps.repository
+
+import scala.collection.concurrent.TrieMap
+
+import rps.model.{GameSummary}
+
+trait GameRepository {
+  def updateGame: GameSummary => Unit
+  def getGame: () => Option[GameSummary]
```

Comment:
This is a strange use of lambdas, why not just

```scala
def updateGate(game: GameSummary): Unit
def getGame(): Option[GameSummary]
```

?

Reply:
much better, I guess I forgot this sintax... :)